### DISCUSSION TITLE
Is there a way to turn off the code actions icon?

### DISCUSSION THREAD

**Discussion by darrenparkinson** [zed-industries/zed#20896] [Feature Requests]
Hi, 

The Code Actions icon is very distracting as it jumps about and flashes when moving lines. Is it possible to keep Code Actions enabled whilst removing that icon?

Many thanks.

https://github.com/user-attachments/assets/9bfbbf47-d12a-442b-ab2f-861f4e6cde58

**Comment by darrenparkinson** [comment]
Found it, sorry.

```json
  "gutter": {
    "code_actions": false
  }
```

**Reply by dusanbrankov** [reply]
This doesn't seem to work anymore. I'm using Zed 0.203.5. The code action button is now placed at the beginning of the buffer and changes position when the cursor is somewhere near of it, which can be very annoying.

From the default settings (`zed: open default settings`), I found the following option to disable it:

```json
{
    "inline_code_actions": true
}
```

**Reply by eldoy** [reply]
Set to false you mean:
```json
{
    "inline_code_actions": false
}
```
